docs/rules/object-property-newline.md

```diff
@@ -72,20 +71,53 @@ Another benefit of this rule is specificity of diffs when a property is changed:
 +var obj = { foo: "foo", bar: "bazz", baz: "baz" };
 ```
 
-### Optional Exception
+### Optional Exceptions
+
+The rule offers three object option.
 
-The rule offers one object option, `allowMultiplePropertiesPerLine`. If you set it to `true`, object literals such as the first two above, with all property specifications on the same line, will be permitted, but one like
+#### `allowMultiplePropertiesPerLine`
+
+If you set `allowMultiplePropertiesPerLine` to `true`, object literals such as the first two above, with all property specifications on the same line, will be permitted, but one like
```

Comment:
In other rule documentation, we like to use patterns like the below:

<pre>
Examples of **correct** code with the `allowMultiplePropertiesPerLine: true` option:

```js
/* examples here */
```

Examples of **incorrect** code with the `allowMultiplePropertiesPerLine: true` option:

```js
/* examples here */
```
</pre>

Referring to other examples ("the first two above") makes the documentation harder to read. It's better to repeat examples than to refer to examples elsewhere in the document.